src/basis/basis.js

```diff
@@ -405,6 +405,9 @@ export class Basis {
                 }
             }
         }
```

Comment:
I don't think we need this if statement anymore once the changes in SOF-5214 are implemented. 

Currently bumping from 0--> 2 stops the updated lattice from being set to 0. `updated lattice = max distance * lattice factor`

SOF-5214 changes this to `updatedLattice = maxDistance + lattice additive factor` so the value should never be 1 it would always be at least the value of the lattice additive factor.